src-tauri/src/command.rs

```diff
@@ -49,6 +47,9 @@ pub async fn handle_action(
                     .ok();
             }
         }
+        ActionType::Unknown => {
+            warn!("received unknown action type with payload `{:?}`", payload);
```

Comment:
We could be a little more verbose on the received unknown action

```suggestion
            warn!("received unknown action type `{:?}` with payload `{:?}`", r#type, payload);
```